web3.md

```diff
@@ -596,6 +598,7 @@ WEB3 JSON RPC
                 <txGasPrice> 1         </txGasPrice>
                 <txNonce>    ACCTNONCE </txNonce>
                 <txGasLimit> 90000     </txGasLimit>
+                <to>         .Account  </to>
```

Comment:
We should already be defaulting to `.Account` from the configuration declaration, do we actually need this?
```suggestion
```